srml/council/src/voting.rs

```diff
@@ -43,20 +43,20 @@ decl_module! {
 
 decl_storage! {
 	trait Store for Module<T: Trait> as CouncilVoting {
-		pub CooloffPeriod get(cooloff_period): required T::BlockNumber;
-		pub VotingPeriod get(voting_period): required T::BlockNumber;
-		pub Proposals get(proposals): required Vec<(T::BlockNumber, T::Hash)>; // ordered by expiry.
-		pub ProposalOf get(proposal_of): map [ T::Hash => T::Proposal ];
-		pub ProposalVoters get(proposal_voters): default map [ T::Hash => Vec<T::AccountId> ];
-		pub CouncilVoteOf get(vote_of): map [ (T::Hash, T::AccountId) => bool ];
-		pub VetoedProposal get(veto_of): map [ T::Hash => (T::BlockNumber, Vec<T::AccountId>) ];
+		pub CooloffPeriod get(cooloff_period) config(): T::BlockNumber = T::BlockNumber::sa(1000);
+		pub VotingPeriod get(voting_period) config(): T::BlockNumber = T::BlockNumber::sa(3);
+		pub Proposals get(proposals) build(|_| vec![0u8; 0]): Vec<(T::BlockNumber, T::Hash)>; // ordered by expiry.
```

Comment:
`vec![u8;0]` should just be `Vec::new()`

Reply:
Due to how I implement this, the closure has to return a specific type, I could have pass down the type info down in the macro, let me check how hard it is.

Reply:
I updated the code to something like this:
```
				$({
					use $crate::codec::Encode;
					let v: $normalbuildty = $normalbuild(&self);
					r.insert(Self::hash(<$normalclassname<$traitinstance>>::key()).to_vec(), v.encode());
				})*
```
by adding the type annotation to `v`, and of course all other related changes. But then there are more issues... errors like:

```
    = note: expected type `<T as Trait>::Hash`
               found type `[u8; 32]`
```

So I'd prefer not change the behavior here.